### ISSUE TITLE
Language settings is not applied. Ubuntu touch

### ISSUE THREAD

**Issue by Easthy** [koreader/koreader#2131]
- KOReader version: 0.0.1 (latest on 1 July 2016, possibly v2015.11-505-g30a9f73)
- Device: BQ Acquaris E5 HD Ubuntu Edition
#### Issue

Language set in language settings is not applied.
#### Steps to reproduce

1) Open language settings
2) Change language
3) Restart app
4) Open setting - language is not changed.

Changed settings
<a href="http://savepic.ru/10372153.htm" target="_blank" ><img src="http://savepic.ru/10372153m.jpg" border="0" alt="Изображение - savepic.ru — сервис хранения изображений" /></a>

After app restart:
<a href="http://savepic.ru/10419256.htm" target="_blank" ><img src="http://savepic.ru/10419256m.jpg" border="0" alt="Изображение - savepic.ru — сервис хранения изображений" /></a>

<a href="http://savepic.ru/10406968.htm" target="_blank" ><img src="http://savepic.ru/10406968m.jpg" border="0" alt="Изображение - savepic.ru — сервис хранения изображений" /></a>

**Comment by Easthy** [comment]
`
-- we can read Lua syntax here!
return {
    ["lastfile"] = "/home/phablet/Documents/Martin_Idien-Dzhiek_London.epub",
    ["night_mode"] = true,
    ["statistics"] = {
        ["is_enabled"] = true,
        ["min_sec"] = 5.0000000000000e+00,
        ["max_sec"] = 9.0000000000000e+01
    },
    ["filemanagermenu_tab_index"] = 3.0000000000000e+00,
    ["lastdir"] = "/home/phablet/Documents",
    ["show_hidden"] = false,
    ["fm_screen_mode"] = "portrait",
    ["kosync"] = {
        ["auto_sync"] = true
    },
    ["language"] = "ru"
}
`

**Comment by chrox** [comment]
This file seems alright, the `language` settings being saved. Did you try setting to other languages? Does any other languages work?

**Comment by Easthy** [comment]
I have tried without success all languages so far.

**Comment by chrox** [comment]
I found that `gettext` looks for `po` files only in the `l10n` directory in current directory (this is true for other platform but on Ubuntu Touch current directory is not the Koreader installation directory thus it fails to find any `po` files).

Ref [this](https://github.com/koreader/koreader/blob/master/frontend/gettext.lua#L48-L49):
``` lua
local file = GetText.dirname .. "/" .. new_lang .. "/" .. GetText.textdomain .. ".po"
local po = io.open(file, "r")
```

**Comment by chrox** [comment]
Could you verify that there are some "cannot open translation file *" strings in the log file?

**Comment by Easthy** [comment]
Yes, I have found theese strings
```
# cannot open translation file l10n/ru/koreader.po
# cannot open translation file l10n/it_IT/koreader.po
# cannot open translation file l10n/cs_CZ/koreader.po
# cannot open translation file l10n/de/koreader.po
# cannot open translation file l10n/fr/koreader.po
# cannot open translation file l10n/nl_NL/koreader.po
```
and so on.

Also you said that log file should be reset every launch. If so I should not see that strings in log. The only one string "cannot open translation file" should be present in the log

**Comment by chrox** [comment]
No, the log is handled by system and won't be reset when it restarts.

**Comment by Frenzie** [comment]
I'll close this one as something that should be quickly noticed and fixed if anyone had a device to work on #4960.
